packages/camera/camera/README.md

```diff
@@ -45,7 +45,10 @@ Change the minimum Android sdk version to 21 (or higher) in your `android/app/bu
 minSdkVersion 21
 ```
 
-It's important to note that the `MediaRecorder` class is not working properly on emulators, as stated in the documentation: https://developer.android.com/reference/android/media/MediaRecorder. Specifically, when recording a video with sound enabled and trying to play it back, the duration won't be correct and you will only see the first frame.
+It's important to note that the endorsed [`camera_android_camerax`][2] implementation of the camera plugin
+built with CameraX has several limitations; please see this list of all of [the list of limitations][3]
```

Comment:
Similar note as on the README: maybe "... with CameraX has better support for more devices, but has some limitations;"